components/base/Typography.tsx

```diff
@@ -113,7 +113,7 @@ const listStyles = css`
   ${baseTextStyles}
   font-size: 16px;
   letter-spacing: 0;
-  line-height: 28px;
+  line-height: 36px;
```

Comment:
Increased line-height of Lists to make it look more close to the mock ups.

Reply:
I agree that this makes the space between bullet points look better, but if an item wraps to multiple lines, it also adds the extra space there, which doesn't look quite right to me (I also found an example of this case in the designs [here](https://projects.invisionapp.com/d/main#/console/19172953/400098958/preview))

Could we change the line-height back and then add the extra space between the bullets by applying a bottom margin to the `<li>`s instead?

Reply:
That's a great point! Didn't even think about that 🤦

Reply:
Address [here](https://github.com/Dwolla/dev-portal/pull/41/commits/d0d3001d409ff380c5db788e1f385048dcd05363). :)